### DISCUSSION TITLE
Settings theme does not look like the same as tutorial

### DISCUSSION THREAD

**Discussion by GronbergInfo** [Textualize/textual#5357] [General]
Hi.

Im currently testing textualize out, i am not sure if i do something wrong, but setting the theme tokyo-night does not make my program look the same as in the [tutorial](https://textual.textualize.io/guide/design/#__tabbed_1_3)

How it looks on my end
![theme](https://github.com/user-attachments/assets/ecbaf5ce-4a1f-4a24-a694-cbcd83c968f1)

My code so far:

```python
#Imports

from textual.app import App, ComposeResult
from textual import events
from textual.widgets import Footer, Header
from textual.binding import Binding

class TSTUI(App):
    COLORS = [
        "white",
        "maroon",
        "red",
        "purple",
        "fuchsia",
        "olive",
        "yellow",
        "navy",
        "teal",
        "aqua",
    ]
    BINDINGS = [
        Binding(key="q", action="quit", description="Quit the app"),
        Binding(
            key="question_mark",
            action="help",
            description="Show help screen",
            key_display="?",
        ),
        Binding(key="delete", action="delete", description="Delete the thing"),
        Binding(key="j", action="down", description="Scroll down", show=False),
    ]

    def on_mount(self) -> None:
        #Background for TSTUI
        #self.screen.styles.background = "navy"
        self.theme = "tokyo-night"

        #Header
        self.title = "Header Application"
        self.sub_title = "With title and sub-title"

    # def on_key(self, event: events.Key) -> None:
    #      if event.key.isdecimal():
    #         self.screen.styles.background = self.COLORS[int(event.key)]

    def compose(self) -> ComposeResult:
        """Create child widgets for the app."""
        yield Header()
        yield Footer()


if __name__ == '__main__':
    print("Loading TUI...")
    app = TSTUI()
    app.run()



```

**Comment by willmcgugan** [comment]
Check you are running the latest version (currently 0.89.1).

**Reply by GronbergInfo** [reply]
# Textual Diagnostics

## Versions

| Name    | Value  |
|---------|--------|
| Textual | 0.89.1 |
| Rich    | 13.9.4 |

**Reply by GronbergInfo** [reply]
# Textual Diagnostics

## Versions

| Name    | Value  |
|---------|--------|
| Textual | 0.89.1 |
| Rich    | 13.9.4 |

## Python

| Name           | Value                      |
|----------------|----------------------------|
| Version        | 3.11.2                     |
| Implementation | CPython                    |
| Compiler       | GCC 12.2.0                 |
| Executable     | /home/dev/venv/bin/python3 |

## Operating System

| Name    | Value                                                |
|---------|------------------------------------------------------|
| System  | Linux                                                |
| Release | 6.1.0-28-amd64                                       |
| Version | #1 SMP PREEMPT_DYNAMIC Debian 6.1.119-1 (2024-11-22) |
## Terminal

| Name                 | Value     |
|----------------------|-----------|
| Terminal Application | *Unknown* |
| TERM                 | xterm     |
| COLORTERM            | *Not set* |
| FORCE_COLOR          | *Not set* |
| NO_COLOR             | *Not set* |

## Rich Console options

| Name           | Value                |
|----------------|----------------------|
| size           | width=161, height=35 |
| legacy_windows | False                |
| min_width      | 1                    |
| max_width      | 161                  |
| is_terminal    | True                 |
| encoding       | utf-8                |
| max_height     | 35                   |
| justify        | None                 |
| overflow       | None                 |
| no_wrap        | False                |
| highlight      | None                 |
| markup         | None                 |
| height         | None                 |


Im running it over SSH, if it matters?

**Comment by GronbergInfo** [comment]
I figured it out.

I was using xterm, did a check in putty and it was using xterm 256 colors, so i changed that in Moba Xterm to use xterm-256 and it now displays the colors as the tutorial.
